src/compiler/generate-xspec-tests.xsl

```diff
@@ -273,6 +279,7 @@
               <xsl:call-template name="x:enter-sut">
                 <xsl:with-param name="instruction" as="element(xsl:sequence)">
                   <sequence>
+                     <xsl:sequence select="x:copy-namespaces($call)"/>
```

Comment:
```suggestion
                    <xsl:sequence select="x:copy-namespaces($call)"/>
```

Reply:
Fixed in 21ed0e1f844324b13670d7ad7828bb9894d2a26f